test cases/failing/46 pkgconfig variables zero length value/meson.build

```diff
@@ -0,0 +1,16 @@
+project('variables-zero-length-test', 'c', version : '1.0')
```

Comment:
Just include both in the same file. If it fails human judgement can understand which part. They are both just testing nearly the same codepath so I just didn't see a need for duplication but not a big deal.